draft-ietf-scitt-architecture.md

```diff
@@ -359,40 +362,40 @@ The SCITT architecture consists of a very loose federation of Transparency Servi
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Signed Inclusion Proofs.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
-This section describes at a high level, the three main roles and associated processes in SCITT: Issuers and the Signed Statement issuance process, Transparency Service and the Signed Statement Registration process, as well as Verifiers of the Transparent Statements and the Receipt validation process.
+This section describes at a high level, the three main roles and associated processes in SCITT: Issuers and Signed Statements, Transparency Service and the Signed Statement Registration process, as well as Verifiers of the Transparent Statements and the Receipt validation process.
 
 ## Signed Statement Issuance and Registration
 
-### Issuer Identity
-
-Before an Issuer is able to produce Signed Statements, it must first create an identifier and obtain an identity document, that is acceptable to the Transparency Service.
-Transparency Services MAY support many different identity document formats.
+### Structure and Content of Signed Statements
 
-Issuers SHOULD use consistent identifiers for all their Statements about Artifacts, to simplify authorization by Verifiers and auditing.
-If an Issuer uses multiple identifiers, they MUST ensure that statements signed under each identifier are consistent.
+Before an Issuer is able to produce Signed Statements, it must first create an identifier and obtain an identity document that is acceptable to the Transparency Service.
```

Comment:
A bit of tweaking as an Issuer can produce signed statements, which they may submit to any number of transparency services. The registration policy may fail if it doesn't support a type of identity, but as worded it sounds like you can't sign a statement, as opposed to the TS doesn't have to accept it.
I was trying to find an alternative way to say this, but is this sentence needed at all?
```suggestion
```

Reply:
With the "new" title and structure I felt that this lead-in sentence now felt off. Less, crisp text is better. Any objections?